lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -179,9 +182,28 @@ export default function BasicLtiLaunchApp() {
   }, [authToken, authUrl, fetchContentUrl, lmsName]);
 
   if (ltiLaunchState.state === 'fetched-url') {
-    return (
-      <iframe width="100%" height="100%" src={ltiLaunchState.contentUrl} />
-    );
+    if (lmsGrader) {
+      // Use the LMS Grader
+      return (
+        <LMSGrader>
+          <iframe
```

Comment:
Is it possible to DRY out this `iframe` between this branch and the `else` branch, as they appear to be identical?